FantasmoSDK/Classes/Models/Analytics/FrameFilterRejectionStatistics.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  FrameFilterRejectionStatistics.swift
+//  FantasmoSDK
+//
+//  Created by Yauheni Klishevich on 26.07.2021.
+//
+
+import Foundation
+
+/// Accumulator of the statistics on the number of the several types of frame filter rejections.
+struct FrameFilterRejectionStatisticsAccumulator {
+    
+    private(set) var filterRejectionReasonToCountDict =
```

Comment:
This is a dictionary of rejection reason counts? I don't think the name needs `To` and also the trailing `Dict` is strange, as we don't typically use a type suffix convention in Swift.

How about just `filterRejectionReasonCounts`?